api/logs/logs.go

```diff
@@ -6,26 +6,50 @@ import (
 	"strings"
 	"time"
 
+	durationUtil "github.com/flanksource/commons/duration"
 	"github.com/flanksource/kommons"
 )
 
 var GlobalBackends []SearchBackend
 
 type SearchConfig struct {
+	// Path is the path of this config file
+	Path     string          `yaml:"-"`
 	Backends []SearchBackend `yaml:"backends,omitempty"`
 }
 
 type SearchBackend struct {
-	Routes     []SearchRoute            `json:"routes,omitempty"`
-	Backend    SearchAPI                `json:"-"`
-	Kubernetes *KubernetesSearchBackend `json:"kubernetes,omitempty"`
-	Files      []FileSearchBackend      `json:"file,omitempty" yaml:"file,omitempty"`
+	Routes        []SearchRoute            `json:"routes,omitempty"`
```

Comment:
I think this should go into a CommonBackend that all backends inline -  Have a route to multiple backends doesn't make much sense.

Reply:
`SearchBackend`, while it has fields for all the backends, it'll only ever be populated by one backend at max. So if I understand it's already like that.

Can you give me an example of how the config file should look like?